action.yml

```
@@ -74,6 +74,10 @@ inputs:
     description: The directory containing compile_commands.json file.
     required: false
     default: ""
+  extra-arg:
+    description: A string of extra arguments passed to clang-tidy for use as compiler arguments.
```

1. This should be a plural form because there's no way to specify this multiple times like clang-tidy and cpp-linter CLI does.
2. We need to discourage the use of spaces unless users are specifying multiple args.
```suggestion
  extra-args:
    description: A string of extra arguments passed to clang-tidy for use as compiler arguments. Multiple arguments are separated by spaces so the argument name and value should use an '=' sign instead of a space.
```